canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -239,26 +239,26 @@ $palette-search-container-height: 41px;
 	display: flex;
 	align-items: center;
 	min-height: 44px;
-	padding-left: 11px;
+	padding-left: $spacing-04;
 	/* Prevent elements from being selectable */
 	/* This stops it being dragged incorrectly.*/
 	user-select: none;
 }
 
 .palette-flyout-category-item-loading {
 	display: flex;
-	margin-left: 16px;
+	margin-left: $spacing-05;
 	width: 100%;
 	height: 100%;
 	justify-content: center;
 	max-height: 60px;
 }
 
 .palette-flyout-category-item-icon {
-	width: 20px;  /* Not specifying height preserves the image's aspect ratio. */
-	margin-left: 5px;
+	width: $spacing-06;  /* Not specifying height preserves the image's aspect ratio. */
```

Comment:
This doesnt look right, you can see the icon size difference from PR vs master:
![palette padding](https://github.com/user-attachments/assets/185329b3-ff84-4c04-b423-357ca246b301)